src/shared/directives/conditional-show/services/condition-parser.service.ts

```diff
@@ -0,0 +1,202 @@
+import { _ as _score } from 'underscore';
+import peg from './condition.peg';
+import { FieldsUtils } from '../../../services/fields/fields.utils';
+
+export class ConditionParser {
+
+    private static readonly CONDITION_NOT_EQUALS = '!=';
```

Comment:
Maybe make the ones in `ShowCondition` public rather than re-declaring them here?

Reply:
as mentioned above made it public in ShowCondition and used it